Octokit/Clients/IOrganizationsClient.cs

```diff
@@ -25,6 +25,12 @@ public interface IOrganizationsClient
         ITeamsClient Team { get; }
 
         /// <summary>
+        /// A client for GitHub's Organization Hooks API.
+        /// </summary>
+        /// <remarks>See <a href="https://developer.github.com/v3/orgs/hooks/">Hooks API documentation</a> for more information.</remarks>
+        IOrganizationHooksClient Hooks { get; }
```

Comment:
This should be in the singular - Hook to be consistent with the other sub-client properties